config/locales/sv.yml

```diff
@@ -539,6 +542,15 @@ sv:
       files_delivery_method: Du har valt att leverera filer via
       method_chosen_when: (vald %{date})
 
+    business_category_display_row:
+      category_no_subcategories_html: Den här kategorin har ingen
+         underkategorier ännu. Du kan begära %{mailto_admin} att tilldela
```

Comment:
Den här kategorin har inga
         underkategorier ännu. Du kan be %{mailto_admin} att lägga till
         en eller flera underkategorier.

Reply:
@thesuss What should "administrator" be (in Swedish)? Or, more precisely, how should we refer to the person/role used for the `mail_to` element?

Here is the code that defines `mailto_admin` in this translation:
```
mail_to_admin = mail_to(ENV['SHF_REPLY_TO_EMAIL'], 'administrator',
                                  subject: t('.subcategories_for',
                                             category_name: business_category.name))
```

I don't see anything in our locale files for "system administrator" (by itself), or "member administrator".

There is this reference to system administrator:

```
payments:
    create:
      page_title: &page_title '%{payment_type} betalning av: %{user_name}'
      title: *page_title
      something_wrong: 'Något gick fel - kontakta systemadministratören på %{admin_email}'
```